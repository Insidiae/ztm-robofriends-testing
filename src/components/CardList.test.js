import React from "react";
import { render } from "@testing-library/react";
import CardList from "./CardList";

describe("<CardList />", () => {
  it("renders properly", () => {
    const mockRobots = [
      {
        id: 1,
        name: "Rob Botterson",
        email: "test@email.com",
      },
    ];
    const { container } = render(<CardList robots={mockRobots} />);

    expect.assertions(1);
    expect(container).toMatchSnapshot();
  });
});
