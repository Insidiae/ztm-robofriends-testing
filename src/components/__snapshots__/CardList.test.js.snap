// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardList /> renders properly 1`] = `
<div>
  <div>
    <div
      class="tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5"
    >
      <img
        alt="robots"
        src="https://robohash.org/1?size=200x200"
      />
      <div>
        <h2>
          Rob Botterson
        </h2>
        <p>
          test@email.com
        </p>
      </div>
    </div>
  </div>
</div>
`;
